src/manager.cpp

```diff
@@ -437,6 +437,9 @@ sdbusplus::async::task<bool>
         case 14: // Error in IPC code
```

Comment:
Commit message:

> "Dec 04 22:44:42 phosphor-rbmc-data-sync-mgr[9200]: Error syncing [/var/lib/ibm/bmcweb/RootCert], ErrCode: 1, Error: rsync: --recurlsive: unknown option",
        "rsync error: syntax or usage error (code 1) at main.c(1782) [client=3.4.1]",
        "RsyncCLI: [RSYNC_CMD]",

Its seems, we have some issue with existing trace i.e I could not see Rsync CLI in the trace `"RsyncCLI: [RSYNC_CMD]`. Please fix it.